cmd/errors.go

```diff
@@ -151,3 +151,8 @@ func isErr(err error, errs ...error) bool {
 	}
 	return false
 }
+
+// Errorf behaves like fmt.Errorf but also traces the returned error.
+func Errorf(format string, args ...interface{}) error {
```

Comment:
We can, but than there's no (real) benefit in using this function at all - We can already use:
`traceError(fmt.Errorf("..."))` I thought: `Errorf` makes it more readable...
Anyway temp. solution because of future namespace separation...